monitor/consul/watcher.go

```diff
@@ -46,16 +46,16 @@ func New(addr, dc, token string, timeout time.Duration, ii ...Item) (*Watcher, e
 	if len(ii) == 0 {
 		return nil, errors.New("items are empty")
 	}
-	if timeout == 0 {
-		timeout = 60 * time.Second
-	}
 	cfg := api.DefaultConfig()
 	cfg.Address = addr
 	var err error
 	cfg.HttpClient, err = api.NewHttpClient(cfg.Transport, cfg.TLSConfig)
 	if err != nil {
 		return nil, err
 	}
+	if timeout > 0 {
+		cfg.WaitTime = timeout
+	}
 	cfg.HttpClient.Timeout = timeout
```

Comment:
This should be deleted as it will rollback your discovery.

Reply:
Oops, forgot to commit this line 🙈